core/form.go

```diff
@@ -30,6 +30,11 @@ type Form struct {
 	// Inline is whether to display the form in one line.
 	Inline bool
 
+	// Modified optionally highlights and tracks fields that have been modified
+	// through an OnChange event. If present, it replaces the default value highlighting
+	// and resetting logic. Ignored if nil.
+	Modified map[string]bool
```

Comment:
We probably want a separate exported bool option for enabling this and then make this unexported?

Reply:
we actually need to set and access the map in plotcore.Editor -- leave as is.